### DISCUSSION TITLE
Multiple Web-Site

### DISCUSSION THREAD

**Discussion by Eyup-Devop** [corazawaf/coraza#959] [Q&A]
Hello,
I have a question, 
Let's say I have 3 websites running on same server and using core rule set but all web sites have different rules too. Is there a way to use same waf engine with core rule set but also custom rules depends on website?
thanks.

**Comment by jptosso** [comment]
Hey!
If it's the same rules you can share the rulesets 
Waf instance is a light object, it won't generate much overhead if you add a few ones, specially if you enable the memoize flag

**Reply by Eyup-Devop** [reply]
thank you.
